packages/fxa-settings/src/components/ExternalAlert/index.test.tsx

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useRef, ReactNode } from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import ExternalAlert from './index';
+import { AlertBarRootAndContextProvider } from '../../lib/AlertBarContext';
+import { MockedCache } from '../../models/_mocks';
+
+describe('ExternalAlert', () => {
+  it('renders as expected', () => {
+    const { rerender } = render(
+      <MockedCache account={{ externalAlert: 'ok' }}>
+        <AlertBarRootAndContextProvider />
+      </MockedCache>
+    );
+    rerender(
+      <MockedCache account={{ externalAlert: 'ok' }}>
+        <AlertBarRootAndContextProvider>
+          <ExternalAlert />
+        </AlertBarRootAndContextProvider>
+      </MockedCache>
+    );
+    expect(screen.getByTestId('alert-bar-root')).toContainElement(
+      screen.getByTestId('alert-bar')
+    );
+    expect(screen.queryByTestId('alert-text')).toBeInTheDocument();
+  });
+
+  it('does not render with no externalAlert text', () => {
+    const { rerender } = render(
+      <MockedCache>
+        <AlertBarRootAndContextProvider />
+      </MockedCache>
+    );
+    rerender(
+      <MockedCache>
+        <AlertBarRootAndContextProvider>
+          <ExternalAlert />
+        </AlertBarRootAndContextProvider>
+      </MockedCache>
+    );
+    expect(screen.queryByTestId('alert-text')).toBeNull();
+  });
+});
```

Comment:
Oh, one additional nice-to-have test here could be that the alert bar shows and then dismisses properly on close since `onDismiss` is handled slightly differently.

Reply:
_squints_ .... meh tbh